cmd/rook/ceph/operator.go

```diff
@@ -45,6 +45,7 @@ func init() {
 
 	operatorCmd.Flags().BoolVar(&csi.EnableRBD, "csi-enable-rbd", false, "enable ceph-csi rbd support")
 	operatorCmd.Flags().BoolVar(&csi.EnableCephFS, "csi-enable-cephfs", false, "enable ceph-csi cephfs support")
+	operatorCmd.Flags().StringVar(&csi.CSIParam.DriverNamePrefix, "csi-driver-name-prefix", "", "custom csi driver name prefix")
```

Comment:
do we need to document it? or update operator-csi.yaml file for this new arg?

Reply:
The default value is fine IMO. We can mention it in the doc but this is really meant for very advanced users and testing.